projects/storefrontlib/cms-components/product/product-list/product-facet-navigation/active-facets/active-facets.component.spec.ts

```diff
@@ -9,7 +9,7 @@ import { By } from '@angular/platform-browser';
 import { RouterTestingModule } from '@angular/router/testing';
 import { Breadcrumb, I18nTestingModule } from '@spartacus/core';
 import { KeyboardFocusModule } from '../../../../../layout/a11y/keyboard-focus/keyboard-focus.module';
-import { EMPTY, of } from 'rxjs';
```

Comment:
Why change this file? This doesn't seem to fit the scope of the PR.
Let's discuss.

Reply:
Don't know why this is here. I didn't make any changes to this file. Maybe during a /develop branch merge it changed these files